modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm

```diff
@@ -183,85 +202,90 @@
 /datum/job/cyborg
 	alt_titles = list(
 		"Cyborg",
-		"Robot",
 		"Android",
+		"Machine",
```

Comment:
I'm not fond of having "Machine" be a choice here, that isn't accurate as to what their function is. They're not a stationary object, they're moving around and performing duties.
```suggestion
```